static/src/javascripts/projects/common/modules/onward/tech-feedback.js

```diff
@@ -116,6 +117,10 @@ define([
      * override those at the time the email is sent.
      */
     return function () {
+        detect.adblockInUse.then(function(adblockInUse){
+            adblockBeingUsed = adblockInUse;
```

Comment:
> your `adblockBeingUsed` var is always going to be `false`

How so? When `detect.adblockInUse` resolves, all the chained promises across all modules will fire synchronously. There is a slight chance, if the user is fast enough (in `ms`), of a false negative.

If we turn this thing async, we'll need to fire `click` events programmatically ... alas, Chrome recently decided not to invoke the default action with "untrusted" events 😩